docs/source/getting_started/demo_script.py

```diff
@@ -58,5 +58,8 @@
     stream=True,
 )
 
+# NOTE: `AgentEventLogger().log(response)` only works with streaming responses.
+# So, if you change the value for `stream` to `False` in the `create_turn` call,
+# comment out the following two lines!
```

Comment:
Why not use a variable for stream and wrap the `for` into a `if` statement instead of this comment?

Reply:
I went for the minimal change, but yes, I've done exactly that in my own "hacks". I'm not sure what the best canonical idiom should be.

Reply:
```diff
diff --git a/docs/source/getting_started/demo_script.py b/docs/source/getting_started/demo_script.py
index 298fd989..a463f9ac 100644
--- a/docs/source/getting_started/demo_script.py
+++ b/docs/source/getting_started/demo_script.py
@@ -52,11 +52,15 @@ agent = Agent(
 prompt = "How do you do great work?"
 print("prompt>", prompt)
 
+use_stream = True
 response = agent.create_turn(
     messages=[{"role": "user", "content": prompt}],
     session_id=agent.create_session("rag_session"),
-    stream=True,
+    stream=use_stream,
 )
 
-for log in AgentEventLogger().log(response):
-    log.print()
+if use_stream:
+    for log in AgentEventLogger().log(response):
+        log.print()
+else:
+    print(response)
```